### ISSUE TITLE
ManagementRestrictionOrRegulationZone - legalBasis - Staging deployment for testing purposes

### ISSUE THREAD

**Issue by DLList** [INSPIRE-MIF/helpdesk-validator#333]
Hello,

I have a problem with the  _Staging deployment for testing purposes of ETF validator_ that I'd like to solve here.

I am currently developing a tool for transforming the data into INSPIRE Area Management/Restriction/Regulation Zones Structures. 

I have created the INSPIRE feature "_ManagementRestrictionOrRegulationZone_" and tested it with the new testing version of the ETF validator: _http://staging-inspire-validator.eu-west-1.elasticbeanstalk.com/etf-webapp/_ .

**My approach for creating INSPIRE-Data**

Creating features "_ManagementRestrictionOrRegulationZone_" and filling the attribute "_legalBasis_" with the voidable value '_unpopulated_' - according to the specification "_legalBasis_" is voidable.
`<am:legalBasis xsi:nil="true" nilReason="http://inspire.ec.europa.eu/codelist/VoidReasonValue/Unpopulated"/>`

**Testing the DataSet with the staging version of Validator**

For the tests I use the test suite "_Conformance class: Application Schema, Area Management, Restriction/Regulation Zones and Reporting Units_".
Here I get an error message - 'am-as.2.1: Constraint for legalBasis':
![grafik](https://user-images.githubusercontent.com/49745407/85024681-b4744f00-b176-11ea-9d3f-f88a88076bc1.png)

In INSPIRE there is actually a constraint that says that _legalBasis_ should be filled in for _ManagementRestrictionOrRegulationZone_ (however, this attribute is also voidable).
INSPIRE-AM-5.3.2.1.1.:
![grafik](https://user-images.githubusercontent.com/49745407/85024756-d1108700-b176-11ea-87c2-e20fb1d71fe7.png)
and
![grafik](https://user-images.githubusercontent.com/49745407/85025213-60b63580-b177-11ea-934f-5cc06ff8397a.png)

**Question**

Is the error message of ETF Validator with the filling of the attribute _legalBasis_ perhaps not correct after all? This attribute is according to the specification voidable.

Best regards,
Dawid

**Comment by fabiovin** [comment]
Dear @DLList,
even if the association has the voidable stereotype, the constraint requires the provision of a legal instrument, therefore the latter must be taken into consideration.
Probably this requirement was encoded in this way in the Implementing rules ("voidable association + constraint" instead of "mandatory association")  in order to better clarify what is required, i.e.  "**the most specific legal instrument** (at least)".

Regards,
Fabio

**Comment by DLList** [comment]
Dear @fabiovin,
thank you very much for clarifying this topic.

Regards,
Dawid

**Comment by JohannaOtt** [comment]
@fabiovin As this is contradictory to what was discussed in the [Thematic Cluster](https://inspire.ec.europa.eu/forum/discussion/view/266637/managementrestrictionorregulationzonecompetentauthority-voidable-but-role-mandatory#elgg-object-266844), I asked for final clarification there.

**Comment by fabiovin** [comment]
Dear @JohannaOtt,
ok, let's continue the discussion on the TC, and then we will report the outcomes here.

**Comment by fabiovin** [comment]
Dear all,
we discussed internally and, even if the constraint is not so clearly expressed, we decided to keep the test less restrictive and to consider the voidable stereotype of the legalBasis association. So, if the legalBasis association is void the validator does not raise an error, if at least one value is present the validator requires a manual check.

This behavior is documented in the [ATS](https://github.com/inspire-eu-validation/data-am/blob/master/am-as/constraints.md) and is already implemented in the [staging instance](http://staging-inspire-validator.eu-west-1.elasticbeanstalk.com/etf-webapp/).

Regards,
Fabio
